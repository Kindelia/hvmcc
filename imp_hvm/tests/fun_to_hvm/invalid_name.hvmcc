function 3haha () {
    {3haha} = Unit
}