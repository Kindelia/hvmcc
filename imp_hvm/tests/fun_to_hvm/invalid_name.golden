Unrecognized token `3` found at 9:10
Expected one of r#"[a-zA-Z_.][a-zA-Z_.0-9]*"#